### PR TITLE
README install guide

### PR DESCRIPTION
**Pull request by viscoze**
## Tension
it's hard to go to packages to find a command to setup a linter

## Solution
move this guides to the top level readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,97 @@ Use robust linting like we do
...
 ## Contributing
 
-It uses [lerna](https://lernajs.io) to maintain all packages within one repository, so take a look at its features. 
+It uses [lerna](https://lernajs.io) to maintain all packages within one repository, so take a look at its features.
+
+## How to install
+
+### Linter for React projects `eslint-config-datarockets`
+
+[![npm version](https://img.shields.io/npm/v/eslint-config-datarockets.svg?style=flat-square)](https://www.npmjs.com/package/eslint-config-datarockets)
```
Comment (dzhlobo) [inline README.md]: Hm... Seems like we have a single version for all the packages: https://github.com/datarockets/style-guide/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R116. So we can remove those badges in each section.

File: README.md [hunk 2]
```diff
@@ -21,7 +21,97 @@ Use robust linting like we do
...
+
+1. Installing deps:
+
+  With `npm`:
+
+  ```
+  (
+    export PKG=eslint-config-datarockets;
+    npm info "$PKG@latest" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG@latest"
+  )
```
Comment (dzhlobo) [inline README.md]: What if we make those installation commands oneliners:
```
export PKG=eslint-config-datarockets; npm info "$PKG@latest" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG@latest"
```

Will it work?

File: README.md (hunk 1 above)
Comment (viscoze) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (viscoze) [reply README.md]: You know, I'd prefer the version with more break lines to show more clearly what this command does.
Something like this:
```
(
  export PKG=eslint-config-datarockets;
  npm info "$PKG@latest" peerDependencies --json \
  | command sed 's/[\{\},]//g ; s/: /@/g' \
  | xargs npm install --save-dev "$PKG@latest"
)
```
How do you feel it?

**Pull request closed without merging**